homeassistant/docker-compose.homeassistant.yml

```diff
@@ -0,0 +1,26 @@
+version: '2'
+
+services:
+  homeassistant:
+    image: 'ghcr.io/home-assistant/home-assistant:${HOME_ASSISTANT_IMAGE_VERSION:-stable}'
+    # devices: # For passing through USB, serial or gpio devices.
+      # - '/dev/ttyUSB0:/dev/ttyUSB0'
+    environment:
+      GUID: 1000
+      PUID: 1000
```

Comment:
**Suggestion:** Consider using environment variables for the GUID and PUID values. This will make your setup more flexible and easier to configure for different environments.
```suggestion
    environment:
      GUID: ${GUID:-1000}
      PUID: ${PUID:-1000}
```